### DISCUSSION TITLE
Wrong human-readable date for commits in PRs

### DISCUSSION THREAD

**Discussion by blablacio** [community/community#126700] [Pull Requests]
### Select Topic Area

Bug

### Body

I just noticed that the human-readable date in this PR is completely wrong. Both commits are from 25.04, definitely not from last week. Am I missing something or this is an actual bug?

<img width="509" alt="image" src="https://github.com/community/community/assets/3009538/65879ab7-1ba2-490f-882e-7e3e441dbf5e">

Link to the PR: https://github.com/cduvray/jwt-authorizer/pull/49

**Comment by daniel-pfeiffer** [comment]
I just noticed the same on [this issue](https://github.com/blackbeam/rust_mysql_common/issues/138): Today, Monday the 17<sup>th</sup>, the last comment from the 7<sup>th</sup> is certainly week before last!

It made me doubt, whether I had reloaded the page. But upon reload, it was again wrongly labelled as last week!
